src/main/java/com/epam/testsystem/github/service/approvement/ApprovementService.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.testsystem.github.service.approvement;
+
+import com.epam.testsystem.github.model.Approvement;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ApprovementService {
+    Approvement add(long userId, long taskId);
+
+    Optional<Approvement> find(long userId, long taskId);
+
+    boolean markGood(long userId, long taskId);
+
```

Comment:
VIEWED - default - after adding. There is no approvement until you click the task.That automatically mean that task VIEWED. If there are't any approvements on task - it has status UNVIEWED.
I prefer zero-arity.